Dockerfile

```diff
@@ -154,6 +154,8 @@ RUN mkdir -p /storage/uploads && \
     chown -Rf www-data.www-data /storage/functions && chmod -Rf 0755 /storage/functions && \
     chown -Rf www-data.www-data /storage/debug && chmod -Rf 0755 /storage/debug
 
+ RUN apk add git
```

Comment:
Why do we add git here? I think it might be a leftover if not its always better to add this kind of tools at the base image level and release a new tag when needed: https://github.com/appwrite/docker-base